### ISSUE TITLE
Enable translations to German in Quality

### ISSUE THREAD

**Issue by dottenbr** [newmatik/Quality#4]
Please enable translations and explain to Carly how she can translate items from English to German.

**Comment by emakis** [comment]
As rmehta responded to my question, as soon we finish the app they will add it to the translation portal. 
https://discuss.erpnext.com/t/translation-for-app/5407/3

Is the app ready enough to ask for that?

**Comment by anandpdoshi** [comment]
You can start with the command: `bench --site [sitename] new-language LANG_CODE APP`

This will create the langcode.csv file in your app

**Comment by dottenbr** [comment]
@emakis please do this and tell Carly what to do!

**Comment by emakis** [comment]
@anandpdoshi, I've created and pushed the translation file. What's next?

**Comment by anandpdoshi** [comment]
You just need to update the CSV file with the translations

Best,
Anand Doshi

ERPNext

**Comment by anandpdoshi** [comment]
You just need to update the CSV file with the translations

Best,
Anand Doshi

ERPNext

**Comment by emakis** [comment]
cvs file sent to Carly

**Comment by dottenbr** [comment]
Not resolved yet, and no process in place for dealing with new translations.

**Comment by emakis** [comment]
the translated file uploaded.
@anandpdoshi how can we create a process to continuously translate the new texts instead of sending and receiveing the csv every time that new texts are added to app?

**Comment by anandpdoshi** [comment]
Install your own translator portal. It is open source.

**Comment by emakis** [comment]
You mean this app https://github.com/frappe/translator ?
And how is it installed? Are there any instruction?

**Comment by anandpdoshi** [comment]
Create a new site, install all apps you want to translate and also install translator. Should do it. Then I think you need to run a command. Check its readme.

**Comment by dottenbr** [comment]
Ideally just install this in the newmatic default site1 so we can translate within the live system?

**Comment by anandpdoshi** [comment]
Translator portal doesn't work like that. It also needs to convert its database into csv files and then be pushed into your app repository. It is just a tool to make mass translation and verification much easier.

**Comment by dottenbr** [comment]
@emakis you have to change the names of the things carly marked in the english original DocType in order for the CSV file to work!

**Comment by emakis** [comment]
What changes? The reds must be deleted? Or what else?

**Comment by dottenbr** [comment]
No, as Carly said the red must be put in and is not part of the doctype
name yet.

2016-02-17 16:04 GMT+01:00 Makis <notifications@github.com>:

> What changes? The reds must be deleted? Or what else?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/newmatik/Quality/issues/4#issuecomment-185243105>.
>



-- 

Mit freundlichen Grüßen / All Best,

*Dominik Ottenbreit*
Geschäftsführer ESO Germany
Director of Business Development

Tel.: ​+49 6752 9112-55
Mobil: ​+49 160 90813466
Fax: ​+49 6752 1524998

www.eso-electronic.com

ESO Electronic Service Ottenbreit GmbH
Hahnenbachstr. 21
55606 Hahnenbach, Germany

Geschaeftsfuehrer (CEO and President): Dominik Ottenbreit
Prokurist (Authorized Signatory):Bjoern Ottenbreit
Gerichtsstand (Jurisdiction): Bad Kreuznach, Germany (HRB 3460)
USt-IdNr. (Tax ID): DE 812 004 832
NCAGE: CH591, DUNS: 314797515, WEEE-Reg-Nr.: DE 84 800 814

**Comment by emakis** [comment]
I suspect that the title doesn't accept parentheses "()" because localy doesn't work anymore. I send a report to support@erpnext and waiting their response. The Quality Report doctype changes. Please check and confirm
